modules/nf-core/blast/makeblastdb/tests/main.nf.test

```diff
@@ -58,4 +58,54 @@ nextflow_process {
 
     }
 
+    test("Should build a blast db folder from a zipped fasta file") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                input[0] = [ file(params.test_data['sarscov2']['genome']['genome_fasta_gz'], checkIfExists: true) ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert process.out.db
+                    with(process.out.db) {
+                        assert size() == 1
+                        with(get(0)) {
+                            File folder = new File(get(0))
+                            File[] listOfFiles = folder.listFiles()
+                            listOfFiles = listOfFiles.sort { it.name }
+                            assert listOfFiles.length == 10
+                            assert listOfFiles[0].name == "genome.fasta"
+                            assert path("${get(0)}/${listOfFiles[0].name}").md5 == "6e9fe4042a72f2345f644f239272b7e6"
+                            assert listOfFiles[1].name == "genome.fasta.gz"
+                            assert listOfFiles[2].name == "genome.fasta.ndb"
+                            assert path("${get(0)}/${listOfFiles[2].name}").md5 == "0d553c830656469211de113c5022f06d"
+                            assert listOfFiles[3].name == "genome.fasta.nhr"
+                            assert path("${get(0)}/${listOfFiles[3].name}").md5 == "f4b4ddb034fd3dd7b25c89e9d50c004e"
+                            assert listOfFiles[4].name == "genome.fasta.nin"
+                            assert listOfFiles[5].name == "genome.fasta.njs"
+                            assert listOfFiles[6].name == "genome.fasta.not"
+                            assert path("${get(0)}/${listOfFiles[6].name}").md5 == "1e53e9d08f1d23af0299cfa87478a7bb"
+                            assert listOfFiles[7].name == "genome.fasta.nsq"
+                            assert path("${get(0)}/${listOfFiles[7].name}").md5 == "982cbc7d9e38743b9b1037588862b9da"
+                            assert listOfFiles[8].name == "genome.fasta.ntf"
+                            assert path("${get(0)}/${listOfFiles[8].name}").md5 == "de1250813f0c7affc6d12dac9d0fb6bb"
+                            assert listOfFiles[9].name == "genome.fasta.nto"
+                            assert path("${get(0)}/${listOfFiles[9].name}").md5 == "33cdeccccebe80329f1fdbee7f5874cb"
+                        }
```

Comment:
This should be a snapshot(Maybe not the gz, nin, and njs files).

Reply:
Done separate snapshots for each file (except empty ones).